src/mbgl/gl/context.hpp

```diff
@@ -124,7 +125,7 @@ class Context final : public gfx::Context {
     State<value::BindFramebuffer> bindFramebuffer;
     State<value::Viewport> viewport;
     State<value::ScissorTest> scissorTest;
-    std::array<State<value::BindTexture>, 2> texture;
+    std::array<State<value::BindTexture>, gfx::MaxActiveTextureUnits> texture;
```

Comment:
lol

Reply:
Yeah that was interesting, I bumped it to 8 which should give plenty of freedom for people implementing custom drawing routines.